apps/whale-api/src/module.api/rawtx.controller.e2e.ts

```diff
@@ -146,3 +147,29 @@ describe('send', () => {
     }
   })
 })
+
+describe('get', () => {
+  it('should accept valid txn and return hex', async () => {
+    const hex = await createSignedTxnHex(container, 10, 9.9999)
+    const result = await controller.send({
+      hex: hex
+    })
+
+    const getResult = await controller.get(result, false)
+
+    expect(hex).toStrictEqual(getResult)
```

Comment:
```suggestion
    const txid = await controller.send({
      hex: hex
    })

    const getResult = await controller.get(txid, false)

    expect(hex).toStrictEqual(getResult)
```

Reply:
yes probably. Sorry for that. If I do more PRs on this repo I need to check my settings.